### ISSUE TITLE
[menu] a11y review of the whole family

### ISSUE THREAD

**Issue by Westbrook** [adobe/spectrum-web-components#1286]
### Feedback

- ~Menu Group needs role="group", not role="presentation"~
    ```
    <span class="header" aria-hidden="true" id="menu-heading-category-1">
        <slot name="header"></slot>
    </span>
    <div role="presentation" aria-labelledby="menu-heading-category-1">
        <slot></slot>
    </div>
    ```
- ~aria-selected does not seem to be reflected when value is set in the first value example~
- ~When value is also used on sp-menu-item it does seem to work, at least for the dropdown case~
- ~When we are rendering a checkbox for the selected element(s) in an action menu item we should use role=menuitemradio when single select or role=menuitemcheckbox when multi-select is enabled. A menuitem cannot take selection.~
- Items not read when navigating using up/down arrow on either Chrome or Safari
- ~Items not read using VO cursor with Chrome. This mode works ok with safari.~

### Path forward

1. ~This structure has changed a little bit post review, and while there _might_ still be a `role="presentation"` groups do need a `role="group"`, likely on the `sp-menu-group` directly...~
2. ~Send selection down to children~
3. Let's review this together. It's unclear what you're seeing.
4. ~Follow the rough standard set by the `sp-action-group` element in this context~
5. Let's review this together. It's unclear what you're seeing.
6. Let's review this together. It's unclear what you're seeing.

cc @jnurthen @adixon-adobe

**Comment by najikahalsema** [comment]
@jnurthen Could you do a new review when you have the time for the `<sp-menu>` and its child components? A lot has changed and we would benefit from new notes. Closing for now.
